virtualizarr/xarray.py

```diff
@@ -137,6 +137,21 @@ def open_virtual_dataset(
         return open_virtual_dataset_from_v3_store(
             storepath=filepath, drop_variables=drop_variables, indexes=indexes
         )
+    elif filetype.name.lower() == "dmrpp":
```

Comment:
Try another syntax for the `FileType` object
```suggestion
    elif filetype == FileType.dmrpp:
```